### DISCUSSION TITLE
Stuck on 3.4.4, later releases fail

### DISCUSSION THREAD

**Discussion by phardy2410** [dougiteixeira/proxmoxve#414] [Q&A] [answered]
Hi 3.4.4 is last version i got working. since then there is a change to the auth method or something and i cannot get it work, i'd appreicate some detailed instructions on what to do in both Proxmox and HA to get later versions to work. thanks.

**Comment by andreasbuff** [comment]
Same to me. Auth with new Token does not work for me either? What do I do wrong?

**Reply by andreasbuff** [reply]
I now can confirm, that 4.0.2 is working. I had to play around with "Username", hence you have to add "@pve" to Username, e.g. "HomeAssistant@pve" and then it is working. Good luck

**Comment by johnfenton1980** [comment]
i followed setup instructions, token does not work ( keep getting unknown error dialog).. but password login does work

**Comment by mZ738** [comment, marked as answer]
The token instructions were the problem, and they are corrected in 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)): a token with **Privilege Separation** on has none of its user's permissions, so the login works and nothing is found, or setup ends in an unknown error. Untick it on the token, or grant the token the same paths and roles as the user; enter the token's name (the part after the `!`) and its secret — pasting the full `user@realm!name` works too. The step-by-step guide is in [Permissions](https://github.com/dougiteixeira/proxmoxve/blob/main/docs/permissions.md). Closing as answered.
